### DISCUSSION TITLE
This is an Slint official example. When no-frame is set to true, the click event will not work. However, when it is set to false, the click event functions correctly

### DISCUSSION THREAD

**Discussion by daydaystudylab** [slint-ui/slint#6753] [Q&A] [answered]
This is an official example. When no-frame is set to true, the click event will not work. However, when it is set to false, the click event functions correctly

```slint
import { Button } from "std-widgets.slint";

export component App inherits Window {

    no-frame: true; //here
    preferred-height: 200px;
    in-out property <int> active-tab;
    VerticalLayout {
        tab_bar := HorizontalLayout {
            spacing: 3px;
            Button {
                text: "Red";
                clicked => { root.active-tab = 0; }
            }
            Button {
                text: "Blue";
                clicked => { root.active-tab = 1; }
            }
            Button {
                text: "Green";
                clicked => { root.active-tab = 2; }
            }
        }
        Rectangle {
            clip: true;
            Rectangle {
                background: red;
                x: root.active-tab == 0 ? 0 : root.active-tab < 0 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
            Rectangle {
                background: blue;
                x: root.active-tab == 1 ? 0 : root.active-tab < 1 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
            Rectangle {
                background: green;
                x: root.active-tab == 2 ? 0 : root.active-tab < 2 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
        }
    }
}

```

**Comment by daydaystudylab** [comment]
The button click event is not working.

**Reply by daydaystudylab** [reply]
I only added one line, no-frame: true, and the click event stopped working. This seems like a bug, but it looks like no one is paying attention to this issue.

**Reply by Vadoola** [reply]
> This seems like a bug, but it looks like no one is paying attention to this issue.

You did post this over the weekend. People may have been busy or not working, and haven't had a chance to respond yet.

Do you have any further details to provide? What Operating System, what version of Slint. How does the bug appear? In SlintPad, in VSCode Preview window, when the program is compiled?

I just tried your code in both SlintPad and on Linux/Wayland with Slint 1.8 and it functioned correctly.

I was going to try on Windows as well, but I upgraded my MS Build tools a couple of days ago, and it looks like I'm getting a bunch of linker errors now, so I can't test that at the moment.

**Reply by daydaystudylab** [reply]
Hello, the relevant information is as follows.

Windows 10

====Cargo.toml=====

[package]
name = "hello_noframe"
version = "0.1.0"
edition = "2021"

[dependencies]
slint = "1.8.0"

[build-dependencies]
slint-build = "1.8.0"

**Reply by daydaystudylab** [reply]
Still, no one is paying attention to this issue 🙂.  As long as no-frame is set to true, the click event will have issues.

**Reply by Vadoola** [reply]
Hi, I can't speak for anyone else, but I haven't been able to recreate the issue. I'm also having some development environment issues preventing me testing it in the exact same environment as you. Mix that with some family health issues, I just haven't had the time to spend on it. Hopefully someone else will chime in soon.

**Comment by hunger** [comment]
This looks like issue #5863. We are stuck on that one apparently :-(

**Comment by Vadoola** [comment, marked as answer]
I'm not sure if this is or is not related to #5863. I managed to get my MS build tools fixed and tried again. The project I created from your sample code works perfectly on my end, so I haven't been able to reproduce this. Do you have any environment variables set, are you making any changes to the Slint backend? Could you create a whole sample project with the issue and post it on github?

**Reply by daydaystudylab** [reply]
import { Button } from "std-widgets.slint";

export component App inherits Window{
    preferred-height: 200px;
    in-out property <int> active-tab;
    no-frame: true;

    
    VerticalLayout {
        tab_bar := HorizontalLayout {
            spacing: 3px;
            Button {
                text: "Red";
                clicked => { root.active-tab = 0; }
            }
            Button {
                text: "Blue";
                clicked => { root.active-tab = 1; }
            }
            Button {
                text: "Green";
                clicked => { root.active-tab = 2; }
            }
        }
        Rectangle {
            clip: true;
            Rectangle {
                background: red;
                x: root.active-tab == 0 ? 0 : root.active-tab < 0 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
            Rectangle {
                background: blue;
                x: root.active-tab == 1 ? 0 : root.active-tab < 1 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
            Rectangle {
                background: green;
                x: root.active-tab == 2 ? 0 : root.active-tab < 2 ? - self.width - 1px : parent.width + 1px;
                animate x { duration: 125ms; easing: ease; }
            }
        }
    }
}

**Reply by daydaystudylab** [reply]
That's all the code

**Comment by greatever350-bit** [comment]
bro did you manage to find a fix? im having this same issue right now..
